docs/manual/expr.xml

```diff
@@ -761,7 +761,8 @@ Require expr replace(%{REQUEST_METHOD},  'E', 'O') == 'GET'"
 Header set foo-checksum "expr=%{md5:foo}"
 
 # This delays the evaluation of the condition clause compared to &lt;If&gt;
-Header always set CustomHeader my-value "expr=%{REQUEST_URI} =~ m#^/special_path\.php$#"
+SetEnvIf REQUEST_URI ^/special_path\.php$ special
+Header always set CustomHeader my-value env=special
 
```

Comment:
The original works fine for me and the replacement doesn't use expressions

Reply:
What I find surprising about the existing example and comment is that delaying the evaluation of REQUEST_URI doesn't really make much sense.  it should really check something like the CONTENT_TYPE or REQUEST_STATUS.

Reply:
There's a different result from the replacement directives?  It seems like the override case is more likely about "onsuccess" vs "always" then about using an expression or an env condition